Chatdown/lib/help.js

```diff
@@ -3,7 +3,7 @@ const chalk = require('chalk');
 const windowSize = require('window-size');
 
 module.exports = function () {
-    process.stdout.write('\nChatdown cli tool used to parse chat dialogs into transcripts\n\n© 2018 Microsoft Corporation\n\n');
+    process.stdout.write('\nChatdown cli tool used to parse chat dialogs (.chat file) into a mock transcript file\n\n© 2018 Microsoft Corporation\n\n');
```

Comment:
Since you mention (.chat file), perhaps do the same for .transcript file?
process.stdout.write('\nChatdown cli tool used to parse chat dialogs (.chat file) into a mock transcript file (.transcript) to be used to view mock dialogs in Bot Emulator?')?